README.md

```diff
@@ -18,11 +22,13 @@ The following directories contain the license list in the format specified:
 
 The file licenses.md is a table of contents for the generated licenses.  The links for the files point to the text for the license.
 
+## Contributing
```

Comment:
I'd rather put this in a `CONTRIBUTING.md` to get [GitHub's increased visibility for that content][1].  We may also want [an issue template][2] with content like:

> Most of this repository is auto-generated.  Please see `CONTRIBUTING.md` for information on the appropriate upstream repositories.

[1]: https://help.github.com/articles/setting-guidelines-for-repository-contributors/#adding-a-contributing-file
[2]: https://help.github.com/articles/creating-an-issue-template-for-your-repository/